app/components/elements/header_component.html.erb

```diff
@@ -0,0 +1 @@
+<%= content_tag variant, value, class: classes %>
```

Comment:
This is simple enough to use `call()` in the component instead of a separate erb.